pub/data/themes.json

```diff
@@ -16,5 +16,11 @@
     "shortName": "csh-datto-bootstrap",
     "cdn": "https://s3.csh.rit.edu/csh-datto-bootstrap/4.0.0/dist/csh-datto-bootstrap.min.css",
     "colour": "5ebbe4"
+  },
+  {
+    "name": "Pastel",
+    "shortName": "csh-pastel-bootstrap",
+    "cdn": "https://s3.csh.rit.edu/csh-datto-bootstrap/4.0.0/dist/csh-datto-bootstrap.min.css",
+    "colour": "F6CECE"
```

Comment:
Would you make the latin characters lowercase to match the rest of the theme colours?